tests/unit/test_compute.py

```diff
@@ -53,6 +56,7 @@
 SLO_CONFIGS_ES = load_slo_samples("elasticsearch", CTX)
 SLO_CONFIGS_DD = load_slo_samples("datadog", CTX)
 SLO_CONFIGS_DT = load_slo_samples("dynatrace", CTX)
+SLO_CONFIG_SPLUNK = load_slo_samples("splunk", CTX)
```

Comment:
`SLO_CONFIGS_SPLUNK`, with an `S` at the end of `CONFIGS` to match the other variables?